libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/AuditEventReturnCode.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.di.ipv.core.library.domain;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+
+import java.util.List;
+
+@ExcludeFromGeneratedCoverageReport
+public record AuditEventReturnCode(
+        @JsonProperty("code") String code, @JsonProperty("issuers") List<String> issuers) {}
```

Comment:
Do you need to specify the @JsonProperty tag if they're the same as the property name?